### PR TITLE
🐛 Fix broken anchor tags in documentation

### PR DESCRIPTION
**Pull request by patrickkettner**

### CODE REVIEW

File: extensions/amp-story/amp-story.md [hunk 1]
```diff
@@ -345,7 +347,7 @@ In most implementations for live blogs, content is either pushed by the server t
 
 This means that publishers of stories do not need to set up a JSON endpoint or push mechanism for this functionality to work.
 
-Content is updated by publishing to the same URL with valid `<amp-story>` markup. The content is pulled into the user's client instance during the next poll. Poll intervals are configurable using the [`data-poll-interval`](#data-poll-interval) attribute.
+Content is updated by publishing to the same URL with valid `<amp-story>` markup. The content is pulled into the user's client instance during the next poll. Poll intervals are configurable using the [`data-poll-interval`](<#live-story-disabled-(optional)>) attribute.
```
Comment (gmajoulet) [inline extensions/amp-story/amp-story.md:350]: This is incorrect, the previous value was right?

File: extensions/amp-story/amp-story.md [hunk 2]
```diff
@@ -355,8 +357,8 @@ As long as the `live-story` attribute is present on the `<amp-story>` element, t
 
 -   Specify an `id` on the `<amp-story>` element.
 -   Add the `live-story` attribute to the `<amp-story>` element.
--   [Optional] Add the [`data-poll-interval`](#data-poll-interval) attribute to the `<amp-story>` element to specify a time interval for checking for new updates.
--   [Optional] When finishing the live broadcast, add the [`live-story-disabled`](#live-story-disabled) attribute to the `<amp-story>` element to disable the polling.
+-   [Optional] Add the [`data-poll-interval`](<#live-story-disabled-(optional)>) attribute to the `<amp-story>` element to specify a time interval for checking for new updates.
```
Comment (gmajoulet) [inline extensions/amp-story/amp-story.md:360]: Same here

File: extensions/amp-story/amp-story.md (hunk 1 above)
Comment (patrickkettner) [reply extensions/amp-story/amp-story.md]: ah! great catch
the existing value was broken, but I duplicated the value being used elsewhere. pushed a fix

File: extensions/amp-story/amp-story.md [hunk 3]
```diff
@@ -345,7 +347,7 @@ In most implementations for live blogs, content is either pushed by the server t
 
 This means that publishers of stories do not need to set up a JSON endpoint or push mechanism for this functionality to work.
 
-Content is updated by publishing to the same URL with valid `<amp-story>` markup. The content is pulled into the user's client instance during the next poll. Poll intervals are configurable using the [`data-poll-interval`](#data-poll-interval) attribute.
+Content is updated by publishing to the same URL with valid `<amp-story>` markup. The content is pulled into the user's client instance during the next poll. Poll intervals are configurable using the [`data-poll-interval`](<#data-poll-interval-(optional)>) attribute.
```
Comment (Enriqe) [inline extensions/amp-story/amp-story.md:350]:
```suggestion
Content is updated by publishing to the same URL with valid `<amp-story>` markup. The content is pulled into the user's client instance during the next poll. Poll intervals are configurable using the [`data-poll-interval`](#data-poll-interval-optional) attribute.
```

File: extensions/amp-story/amp-story.md [hunk 4]
```diff
@@ -355,8 +357,8 @@ As long as the `live-story` attribute is present on the `<amp-story>` element, t
 
 -   Specify an `id` on the `<amp-story>` element.
 -   Add the `live-story` attribute to the `<amp-story>` element.
--   [Optional] Add the [`data-poll-interval`](#data-poll-interval) attribute to the `<amp-story>` element to specify a time interval for checking for new updates.
--   [Optional] When finishing the live broadcast, add the [`live-story-disabled`](#live-story-disabled) attribute to the `<amp-story>` element to disable the polling.
+-   [Optional] Add the [`data-poll-interval`](<#data-poll-interval-(optional)>) attribute to the `<amp-story>` element to specify a time interval for checking for new updates.
```
Comment (Enriqe) [inline extensions/amp-story/amp-story.md:360]:
```suggestion
-   [Optional] Add the [`data-poll-interval`](#data-poll-interval-optional) attribute to the `<amp-story>` element to specify a time interval for checking for new updates.
```

File: extensions/amp-story/amp-story.md [hunk 5]
```diff
@@ -355,8 +357,8 @@ As long as the `live-story` attribute is present on the `<amp-story>` element, t
...
+-   [Optional] When finishing the live broadcast, add the [`live-story-disabled`](<#live-story-disabled-(optional)>) attribute to the `<amp-story>` element to disable the polling.
```
Comment (Enriqe) [inline extensions/amp-story/amp-story.md:361]:
```suggestion
-   [Optional] When finishing the live broadcast, add the [`live-story-disabled`](#live-story-disabled-optional) attribute to the `<amp-story>` element to disable the polling.
```

**Review by Enriqe — COMMENTED** [review]
Thank you very much for fixing these!! 

Just some things I recently learned: whenever you see `<#...>` it means prettier found a broken link in the markdown and needs to be fixed. Also parenthesis should be removed from the link on e.g. `#data-poll-interval-(optional)` -> `#data-poll-interval-optional`

File: extensions/amp-story/amp-story.md (hunk 3 above)
Comment (patrickkettner) [reply extensions/amp-story/amp-story.md:350]: ah! thank you for the explanation on the angle brackets

on amp.dev, the parens are not being removed in the html, and as a result the anchors aren't working without them (e.g. https://amp.dev/documentation/components/amp-story/#poster-square-src-optional vs https://amp.dev/documentation/components/amp-story/#poster-square-src-(optional))

**Review by gmajoulet — APPROVED** [review]
`/amp-story/**` LGTM, cc @CrystalOnScript for the overall approval :))

**Review by CrystalOnScript — APPROVED** [review]

**Comment by CrystalOnScript** [comment]
This looks great, thank you!

File: extensions/amp-story/amp-story.md (hunk 3 above)
Comment (Enriqe) [reply extensions/amp-story/amp-story.md:350]: Won't they still be broken if they have the `<>` surrounding the links? even in amp.dev?

Comment (patrickkettner) [reply extensions/amp-story/amp-story.md:350]: no, it is functional with them, and the build won't pass with prettifier failures

**Pull request merged**